proc_Hahn_echo.py

```diff
@@ -2,31 +2,26 @@
 from scipy.optimize import leastsq,minimize,basinhopping
 fl = figlist_var()
 for date,id_string in [
-        ('200302','alex_probe_water'),
+        ('201211','Ni_sol_probe_2'),
         ]:
     filename = date+'_'+id_string+'.h5'
     nodename = 'signal'
     s = nddata_hdf5(filename+'/'+nodename,
             directory = getDATADIR(exp_type = 'test_equip'))
-    print(ndshape(s))
     nEchoes = s.get_prop('acq_params')['nEchoes']
     SW_kHz = s.get_prop('acq_params')['SW_kHz']
     nPoints = s.get_prop('acq_params')['nPoints']
     s.set_units('t','s')
-    print((s.get_prop('acq_params')))
-    print((s.get_prop('nScans')))
-    fl.next('raw data')
-    fl.plot(s.real,alpha=0.4)
-    #fl.plot(s.imag,alpha=0.4)
-    #fl.plot(abs(s),':',c='k',alpha=0.4)
+    fl.next('raw data-real')
+    fl.plot(s.real,alpha=0.4,label='real')
+    fl.plot(s.imag,alpha=0.4,label='imaginary')
     s.ft('t',shift=True)
+    s.mean_all_but('t')
     s.convolve('t',10)
-    fl.next('Absolute value -- FT')
```

Comment:
If you're going to do a convolution, you should mention it in the plots.  I'm also not a fan of doing a convolution like this -- rather choose your apodization carefully in the time domain.  We sometimes convolve right when plotting, but that's a convenience thing that has to do w/ the display, and isn't intended as a permanent code alteration, like here.